charmcraft/main.py

```diff
@@ -240,10 +240,10 @@ def _add_global_options(self, parser, context):
         mutexg = parser.add_mutually_exclusive_group()
         mutexg.add_argument(
             '-v', '--verbose', action='store_true', dest='verbose_' + context,
-            help="be more verbose and show debug information")
+            help="More verbose and show debug information.")
```

Comment:
I think you wanted the 'be' there; you wanted it in imperative, right?
```suggestion
            help="Be more verbose and show debug information.")
```

OTOH this text makes it a little unclear: is being more verbose a separate thing from showing debug information, or are you explaining what being more verbose means? To disambiguate, and assuming (as I understand it) that you want to mean the second one, you could expand it a bit: e.g. “Be more verbose, and also show debug information” or perhaps even “Be more verbose, as well as showing debug information” (this last one trips me up so it can probably be improved). Or maybe “Show debug information, and be more verbose.” might do the trick.

Reply:
Going for "Show debug information, and be more verbose"